packages/integration/src/azure/AzureUrl.ts

```diff
@@ -37,6 +37,10 @@ export class AzureUrl {
       owner = parts[1];
       project = parts[2];
       repo = parts[4];
+    } else if (parts[4] === '_git') {
+      owner = `${parts[1]}/${parts[2]}`;
```

Comment:
What does the owner actually refer to in this case? as in what is part 1 and 2? Looks a bit risky to do this concatenation.

Reply:
older TFS instances that has been upgraded over time then you may have it setup under the /tfs subpath in the URL which is what is the issue here

Reply:
example ......  tfs/projects/